tests/conftest.py

```diff
@@ -762,11 +773,14 @@ def setup_func(self, module_dir: Optional[str]):
         self._keep = False
         self.project_dir = tempfile.mkdtemp()
         self.modules_dir = module_dir
+        self.process_env_backup = ProcessEnv.python_path
```

Comment:
Why is this here? (i.e. why is it not in the global reset, like the other venv related stuff?)

Reply:
Good point, I should probably move this.

Reply:
No reason, it would be better placed in the global reset fixture, I've moved it there.